### PR TITLE
Add test project for JDK 18

### PR DESCRIPTION
**Pull request by jgrassel**
Add test case project for JDK 18

### CODE REVIEW

**Review by gjwatts — COMMENTED** [review]
Please see my review comments

File: io.openliberty.java.internal_fat_18/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Repo for quick testing of Liberty apps
+Especially useful for new Java versions
+
+### For versions of Java that already have gradle support 
+
+To build the WAR file locally, from the root directory of the open-liberty-misc repository run:
+
+```
+./gradlew io.openliberty.java.internal_fat_17:build
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply io.openliberty.java.internal_fat_18/README.md:9]: huh.  I thought I got them all, kind of surprised I missed that.

[the remark this answers is not in the document]
Comment (jgrassel) [reply io.openliberty.java.internal_fat_18/README.md]: Done

File: io.openliberty.java.internal_fat_18/build.gradle [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+dependencies {
+    compileOnly group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.1.1'
+//    compileOnly group: 'javax.persistence', name: 'javax.persistence-api', version: '2.2'
+    compileOnly group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
+    compileOnly group: 'javax.enterprise', name: 'cdi-api', version: '2.0'
+//    compileOnly group: 'javax.annotation', name: 'javax.annotation-api', version: '1.3.2'
+//    compileOnly group: 'javax.transaction', name: 'javax.transaction-api', version: '1.3'
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '21.0.0.9'
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply io.openliberty.java.internal_fat_18/build.gradle]: Done

File: io.openliberty.java.internal_fat_18/src/main/java/io/openliberty/java/internal/TestApp.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply io.openliberty.java.internal_fat_18/src/main/java/io/openliberty/java/internal/TestApp.java]: Done

File: io.openliberty.java.internal_fat_18/src/main/java/io/openliberty/java/internal/TestService.java [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            e.printStackTrace(System.out);
+            e.printStackTrace(new PrintWriter(sw));
+            log("<<< EXIT FAILED");
+        }
+        String result = sw.toString();
+        sw = new StringWriter();
+        return result;
+    }
+
+    private void doTest() throws Exception {
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply io.openliberty.java.internal_fat_18/src/main/java/io/openliberty/java/internal/TestService.java:46]: Done

File: io.openliberty.java.internal_fat_18/.settings/org.eclipse.wst.common.component [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project-modules id="moduleCoreId" project-version="1.5.0">
+	<wb-module deploy-name="io.openliberty.java.internal_fat_17">
+		<property name="context-root" value="io.openliberty.java.internal_fat_17"/>
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply io.openliberty.java.internal_fat_18/.settings/org.eclipse.wst.common.component]: Done

File: io.openliberty.java.internal_fat_18/.settings/org.eclipse.wst.common.project.facet.core.xml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<faceted-project>
+  <runtime name="Getting started Liberty Server"/>
+  <fixed facet="jst.java"/>
+  <fixed facet="jst.web"/>
+  <installed facet="jst.java" version="17"/>
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply io.openliberty.java.internal_fat_18/.settings/org.eclipse.wst.common.project.facet.core.xml]: Done

File: io.openliberty.java.internal_fat_18/src/main/java/module-info.java [hunk 7]
```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply io.openliberty.java.internal_fat_18/src/main/java/module-info.java]: Done

**Review by gjwatts — APPROVED** [review]

**Pull request merged**